app/api/admin_translations.py

```diff
@@ -0,0 +1,22 @@
+from flask import send_file, Blueprint
+import os
+import shutil
+import uuid
+import tempfile
+
+app_dir = os.path.dirname(os.path.abspath(os.path.dirname(__file__)))
+file_dir = tempfile.mkdtemp()
+admin_blueprint = Blueprint('admin_blueprint', __name__, url_prefix='/admin/content/translations/all')
+uuid_literal = uuid.uuid4()
+temp_dir = tempfile.mkdtemp()
+
+@admin_blueprint.route('/', methods=['GET'])
+def download_translations():
+    """Admin Translations Downloads"""
+    shutil.make_archive('translations{}'.format(uuid_literal), 'zip', temp_dir)
+    # shutil.move('translations{}.zip'.format(uuid_literal), app_dir)
+    from .helpers.tasks import delete_translations
+    delete_translations.apply_async(temp_dir, countdown=600)
```

Comment:
You should not delete the entire directory. Only the file

Reply:
Alright. Replaced the temp_dir with filename